tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -22,13 +24,22 @@ def get_arguments():
     2) a "class file" (newline delimited list of strings that correspond to RDF classes)
     """
     parser = argparse.ArgumentParser(description='Process data model export')
-    parser.add_argument('-f', '--file_path', help="a path to the data model", required=True)
+    parser.add_argument('-f', '--file_path', help="a path to the data model")
     # TODO: two methods to input classes, CLI list vs CLI file_path argument - keep both?
     parser.add_argument('-l', '--class_list', nargs='+', help="a class listing string")
     parser.add_argument('-c', '--class_path', help="a class listing file")
     args = parser.parse_args()
 
-    if args.class_list is None and args.class_path is None:
+    if args.file_path is None and args.class_list is None and args.class_path is None:
```

Comment:
🔧 I think rather than dropping a log statement here, we can lean on argparse's default help handling and improve the help text on the args if need be.